### PR TITLE
Include `repmgr node check` in healthcheck output.

### LINKED ISSUE #102: Consider leveraging repmgr's built-in health checks
```
repmgr node check

Node "fdaa:0:2e26:a7b:c850:fa7a:5336:2":
	Server role: OK (node is primary)
	Replication lag: OK (N/A - node is primary)
	WAL archiving: OK (0 pending archive ready files)
	Upstream connection: OK (N/A - node is primary)
	Downstream servers: OK (2 of 2 downstream nodes attached)
	Replication slots: OK (2 of 2 physical replication slots are active)
	Missing physical replication slots: OK (node has no missing physical replication slots)
	Configured data directory: OK (configured "data_directory" is "/data/postgresql")
```

### PR DESCRIPTION
**Pull request by DAlperin**
Closes #102

### CODE REVIEW

File: internal/flycheck/pg.go [hunk 1]
```diff
@@ -55,6 +59,14 @@ func CheckPostgreSQL(ctx context.Context, checks *check.CheckSuite) (*check.Chec
 	return checks, nil
 }
 
+func repmgrCheck() (string, error) {
+	cmd, err := utils.RunCommand("repmgr node check --config-file /data/repmgr.conf", "postgres")
```
Comment (davissp14) [inline internal/flycheck/pg.go:63]: We should pull the `--config-file` value from `node.RepMgr.ConfigPath` instead of hard-coding it.

Comment (davissp14) [reply internal/flycheck/pg.go:63]: nit: `cmd` seems like a weird variable name. :D

**Review by davissp14 — APPROVED** [review]
Few comments, but looks good!

File: internal/flycheck/pg.go (hunk 1 above)
Comment (guillaumervls) [reply internal/flycheck/pg.go:63]: > We don't have a great way to access this from the healthcheck server
@DAlperin is this blocking?

**Comment by davissp14** [comment]
Re-open if you decide to pick this back up.

**Pull request closed without merging**